src/config.c

```diff
@@ -523,7 +523,10 @@ void merge_config(struct vpn_config *dst, struct vpn_config *src)
 		free(dst->pinentry);
 		dst->pinentry = src->pinentry;
 	}
-	if (src->realm[0])
+        if (src->tun != invalid_cfg.tun) {
+                dst->tun = src->tun;
+        }
+        if (src->realm[0])
```

Comment:
Looks good, please fix indentation.

Reply:
ups what did i miss, thought i had them all on 8 spaces as required ?

Reply:
It's tabs, not spaces.

Reply:
Looks good now!